packages/terra-form-select/src/search/Frame.jsx

```diff
@@ -442,7 +442,7 @@ class Frame extends React.Component {
       searchValue: '',
       hasSearchChanged: false,
       isOpen: false,
-      isAbove: false,
+      // isAbove: false,
```

Comment:
you could remove this line. if not required.

Reply:
updated here https://github.com/cerner/terra-core/pull/2904/commits/2d8de879a0187eb87b5dea6d6877d871738c1ef4